catalog/service/common/src/main/java/org/projectnessie/catalog/service/objtypes/EntitySnapshotObj.java

```diff
@@ -68,7 +68,7 @@ default ObjType type() {
           TaskObj.taskDefaultCacheExpire(),
           c -> ObjType.NOT_CACHED);
 
-  static ObjId snapshotIdFromContent(Content content) {
+  static ObjId snapshotObhIdFromContent(Content content) {
```

Comment:
Small typo:

```suggestion
  static ObjId snapshotObjIdFromContent(Content content) {
```

Also, this method name uses "from" while the `entityObjIdForContent` method uses "for". Would be nice to use one or the other consistently.